src/stdlib.sol

```diff
@@ -11,12 +11,12 @@ abstract contract stdCheats {
     // Setup a prank from an address that has some ether
     function hoax(address who) public {
         vm_std_cheats.deal(who, 1 << 128);
-        vm_std_cheats.prank(who);
+        vm_std_cheats.prank(who, who);
```

Comment:
default shouldnt be to override origin. We can overload these as (address, address) or (address, address, uint256) for the give example